spec/yaml/swiftnav/sbp/ndb.yaml

```diff
@@ -11,7 +11,7 @@
 package: swiftnav.sbp.ndb
 description: >
   Messages for logging NDB events.
-stable: False
+stable: True
```

Comment:
If we're being conservative I would vote to leave these and not stable. I'm not sure what their use is, or even if Piksi Multi produces them.